src/test/java/com/mael/ttt/ui/UserInterfaceTest.java

```diff
@@ -69,8 +71,8 @@ public void readsUserInput() {
 
     @Test
     public void printsWinnerMessage() {
-        ui.printHasWinnerMessage("foo");
-        assertEquals(UserInterface.HASWINNER + "foo", spy.firstPrintedMessage());
+        ui.printHasWinnerMessage(PLAYER);
+        assertEquals(UserInterface.HASWINNER + "X", spy.firstPrintedMessage());
```

Comment:
This couples the ui representation to your current symbol.
Instead of `"X"` maybe a `PLAYER.getString()` makes sure this test does not break just because you change your UI from "X" to an ":x:" (or "A", etc.).